### PR TITLE
Add tests for console split panel

### PR DESCRIPTION
**Pull request by jasonLaster**
This is ready to be merged once the [m-c patch](https://bugzilla.mozilla.org/show_bug.cgi?id=1302202) lands.

### CODE REVIEW

File: public/js/components/App.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ const App = React.createClass({
   },
 
   onKeyDown(e) {
-    if (e.key === "Escape") {
+    if (this.state.searchOn && e.key === "Escape") {
```
Comment (jasonLaster) [inline public/js/components/App.js]: this is needed to let the escape keydown bubble up to the document

**Comment by jasonLaster** [comment]
@jlongster mind pushing the console patch when you have a moment

**Comment by jlongster** [comment]
Yeah, I pushed it last night and it landed on m-c already.

**Comment by jasonLaster** [comment]
@jlongster i saw that, thanks!

**Comment by jasonLaster** [comment]
@jlongster let me know when the docker image is ready and i can kick circle

**Pull request merged**